packages/rspack/src/util/index.ts

```diff
@@ -1,5 +1,4 @@
 import terminalLink from "terminal-link";
-import isCI from "is-ci";
```

Comment:
Please remove this package as well ;-).

Reply:
Would you also migrate the configuration to the newest version? This is what the issue means https://github.com/web-infra-dev/rspack/issues/5067.

Reply:
is-ci existed before, in the prepare field in the script in package.json under the root directory。 i just add @type/is-ci